src/test/java/org/terasology/logic/health/HealEventTest.java

```diff
@@ -38,7 +39,6 @@
 public class HealEventTest extends ModuleTestingEnvironment {
 
     private EntityManager entityManager;
-    private Time time;
 
     private static final Logger logger = LoggerFactory.getLogger(HealEventTest.class);
```

Comment:
And this line could be moved above the `entityManager` to resolve a variable order issue